x11-plugins/pidgin-telegram/pidgin-telegram-1.3.1.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="A libpurple protocol plugin that adds support for the Telegram messenger"
+HOMEPAGE="https://github.com/majn/telegram-purple"
+SRC_URI="https://github.com/majn/telegram-purple/releases/download/v${PV}/telegram-purple_${PV}.orig.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
```

Comment:
Incorrect, please read the copyright notices in files.

Reply:
Changed to `GPL-2+`